macros/src/command/mod.rs

```diff
@@ -49,6 +49,8 @@ pub struct CommandArgs {
     category: Option<String>,
     custom_data: Option<syn::Expr>,
 
+    manual_cooldowns: bool,
```

Comment:
Yeah I'd vouch for `Option<bool>`. Negligible extra complexity imo (especially not if you just use the attribute macro API, since it makes no difference there), but keeps the door open for more funky use cases users will think of

Reply:
Resolved.